### PR TITLE
Update Book.java

### PR DESCRIPTION
**Pull request by tomarru**

### CODE REVIEW

**Comment by tomarru** [comment]
@tassadar2 Please provide a feedback for my change

File: src/main/java/com/github/demo/model/Book.java [hunk 1]
```diff
@@ -41,6 +43,15 @@ public String getAuthor() {
     public void setAuthor(String author) {
         this.author = author;
     }
+    
+    public String getFeedback(){
+        return feedback;
+    }
+   
+    public void setFeedbacks(String feedback){
```
Comment (tassadar2) [inline src/main/java/com/github/demo/model/Book.java]: not following code standard

File: src/main/java/com/github/demo/model/Book.java [hunk 2]
```diff
@@ -41,6 +43,15 @@ public String getAuthor() {
...
+
```
Comment (tassadar2) [inline src/main/java/com/github/demo/model/Book.java]: too much whitespace

File: src/main/java/com/github/demo/model/Book.java [hunk 3]
```diff
@@ -53,4 +64,5 @@ public String getCover() {
     public void setCover(String cover) {
         this.cover = cover;
     }
+
```
Comment (tassadar2) [inline src/main/java/com/github/demo/model/Book.java:66]: mix of functionality and layout

File: src/main/java/com/github/demo/model/Book.java [hunk 4]
```diff
@@ -41,6 +43,15 @@ public String getAuthor() {
...
         this.author = author;
     }
+    
+    public String getFeedback(){
+        return feedback;
+    }
+   
+    public void setFeedbacks(String feedback){
+        
+        this.feedback=feedback;
```
Comment (tassadar2) [inline src/main/java/com/github/demo/model/Book.java]: oh oh, code layout again

**Review by tassadar2 — COMMENTED** [review]
this was pretty great! keep up the good work pls

File: src/main/java/com/github/demo/model/Book.java (hunk 3 above)
Comment (tomarru) [reply src/main/java/com/github/demo/model/Book.java:66]: @tassadar2 Done!

**Review by tomarru — COMMENTED** [review]
Submitting my changes

**Pull request merged**